client/Pages/EventTypeCreate/Update.elm

```diff
@@ -310,6 +315,10 @@ submitCreate model =
             , ( "category", asString FieldCategory )
             , ( "partition_strategy", asString FieldPartitionStrategy )
             , ( "partition_key_fields", partitionKeyFields )
+            , ( "ordering_key_fields", orderingKeyFields )
+            , ( "compatibility_mode", asString FieldCompatibilityMode )
```

Comment:
Why is it needed to add `compatibility_mode` in this PR?

Reply:
good catch! some merge bug